kytos/core/api_server.py

```diff
@@ -61,6 +62,22 @@ def __init__(self, app_name, listen='0.0.0.0', port=8181,
         # Update web-ui if necessary
         self.update_web_ui(force=False)
 
+        @self.app.errorhandler(HTTPException)
+        def handle_exception(e):
```

Comment:
You can maybe change the variable name from `e` to a more explicit name as `http_error` or `err`. This approach avoids the warning of the invalid-name.

Reply:
Variable name changed.